tests/can_gateway/Makefile

```diff
@@ -0,0 +1,65 @@
+include ../Makefile.tests_common
+
+BOARD ?= native
+
+RIOTBASE ?= $(CURDIR)/../..
+
+USEMODULE += shell
+USEMODULE += shell_commands
+
+USEMODULE += auto_init_can
+USEMODULE += conn_can
+USEMODULE += can_trx
+
+FEATURES_REQUIRED += periph_can
+FEATURES_REQUIRED += periph_gpio_irq
+
+USEMODULE += schedstatistics
+
+TRX_TO_BUILD ?= ncv7356
```

Comment:
Isn't this a very hardware-specific thing to include here? 
I mean, usually a CAN-transceiver (such as the MCP2551, TJA1054 or the SN65HVD233) works passively, without having any control signals, and thus the software does not need to know what transceiver is connected. 
This ncv7356 appears to be an exception?

Reply:
the NCV7356 transceiver requires a driver to work correctly, which is available in RIOT.